documents/minutes/2019-08-06 WCA Board Meeting.md

```diff
@@ -0,0 +1,87 @@
+<div class="text-center">
+MINUTES OF MEETING OF THE BOARD OF DIRECTORS OF WORLD CUBE ASSOCIATION
+
+A California Nonprofit Public Benefit Corporation
+</div>
+
+On August 6, 2019 at 9:00 a.m. PST, the Board of Directors of World Cube Association (the “Board”) met for a regular meeting via Google Hangouts. The following directors were present at the meeting:
+
+<div class="text-center">
+Alberto Pérez de Rada Fiol <br>
+Christopher Wright <br>
+Ethan Pride <br>
+Robert Burton <br>
+</div>
+
+<b class="text-center">BOARD MEETING FREQUENCY</b>
+
+The Board agreed to schedule quarterly Board meetings. Attendance at such meetings shall be subject to the WCA Board Attendance Policy.
+
+<b class="text-center">MAJOR CHAMPIONSHIPS</b>
+
+As a result of negative feedback regarding the timing for applications with the current round of Continental Championships, in conjunction with feedback from Ethan Pride regarding the difficulty of planning the World Championship in less than one year, the Board agreed to open application periods earlier and extend their length to offer organizational teams more time to apply and give the successful applicants more time to prepare. The following timetable was agreed upon:
+
+| **Type** | **Year** | **Location** | **Applications open** | **Applications close** | **Board decision** |
+| :--------- | :--------- | :--------- | :--------- | :--------- | :--------- |
+| World | 2021 | World | January 1, 2020 | March 31, 2020 | April 30, 2020 |
+| Continental | 2022 | Asia | March 1, 2020 | May 31, 2020 | June 30, 2020* |
+| | | Europe | | | |
```

Comment:
I believe markdown gives you the option to put whitespace in here to make this more readable, if you want. It's more of a hassle, and it will render the same either way, so it's really just a "how readable do you want the source code to be", sort of thing.

Reply:
That's good to know! Although I guess it's not very important in this case given that this will likely never have to be edited.